Makefile

```diff
@@ -18,7 +18,7 @@ ifneq ($(UNAME), Darwin)
 CXX_FLAGS += -fPIC
 endif
 
-LINK_FLAGS = -L ./lib -L $(OPT_LIB_PATH) -L $(ISL_PATH) -lclkwrk -lbarvinok -lisl -lntl -lgmp -lpolylibgmp -lpthread 
+LINK_FLAGS = -L ./lib -L $(OPT_LIB_PATH) -L $(ISL_PATH) -lclkwrk -lbarvinok -lisl -lntl -lgmp -lpolylibgmp -lpthread -lstdc++fs
```

Comment:
What is this link flag and why is it needed?

Reply:
This is the c++ filesystem library. It helps me to grep all files in the verilog directory and link in the verilog command. 
I create this util function using that library. 
https://github.com/dillonhuff/clockwork/blob/c385626aa9ba5c022c09aed07aa0933638ca0c85/utils.cpp#L3-L13